bin/mail/2016-09-27-latam-hoc-minecraft/generate-mailing-lists.rb

```diff
@@ -0,0 +1 @@
+../2016-09-15-latam-hoc-starwars/generate-mailing-lists.rb
```

Comment:
nit: Please add a comment in the referenced file that it is also being used in this mailing. Generally, I think people view these files as being independent, it is probably worth calling out when they are not.